src/DynamoCoreWpf/ViewModels/Core/StateMachine.cs

```diff
@@ -152,8 +153,15 @@ internal void UpdateDraggedSelection(Point2D mouseCursor)
                     "UpdateDraggedSelection cannot be called now");
             }
 
-            foreach (DraggedNode draggedNode in draggedNodes)
-                draggedNode.Update(mouseCursor);
+            var parentAnnotation = DynamoSelection.Instance.Selection.OfType<AnnotationModel>().FirstOrDefault(x => x.HasNestedGroups);
```

Comment:
you cannot select two annotations with nested groups?

Reply:
Yes we can. 
I will update the PR to handle that scenario